### DISCUSSION TITLE
Reset password doesn't seem to be working

### DISCUSSION THREAD

**Discussion by akramcshive** [hagopj13/node-express-boilerplate#128] [Q&A]
So i have been trying reset password endoint from postman and for some reason the reset password is always failing saying password reset failed. 

Any help with the request headers or how the flow is would be of great help.

**Comment by hagopj13** [comment]
Please refer to the validation of `reset-password` endpoint https://github.com/hagopj13/node-express-boilerplate/blob/b20899c15b90a2d4d2654b5b7684fcbdc8676989/src/validations/auth.validation.js#L37

Note that, the token is generated using the `forgot-password` endpoint
